docs/pages/index.mdx

```diff
@@ -0,0 +1,124 @@
+# MUD
```

Comment:
For now this file is just a copy of the root level README.md. Soon we'll have separate specific front page for mud.dev.

Reply:
I didn't find an option to specify a default theme in the docs, i guess we could hack it by eg. setting the `theme` key in the local storage, but we could also just wait to publish the nextra docs under [mud.dev](https://mud.dev) until we have the new design (right now they're under [nextra.mud.dev](https://nextra.mud.dev))